modules/ROOT/pages/install-checklist.adoc

```diff
@@ -134,6 +134,25 @@ These are the requirements:
 * Access and connectivity to private container registry
 * For OpenShift use SecurityContextConstraints for installing Anypoint Platform PCE.
 
+=== Node Requirements for Anypoint Platform PCE 4.1
+
+Before executing the `pcectl` binary for Anypoint Platform PCE 4.1, ensure you follow these requirements:
+
+* Use a Linux based operating system
```

Comment:
We should add another requirement for gpg, like: Use gpg (GnuPG) version 2.1 or higher. Check the version by running: gpg --version